VerilogCodingStyle.md

```diff
@@ -1756,6 +1756,18 @@ an always block as occurring in a separate simulation event as the non-blocking
 assignment. This process makes some signals jump registers, potentially leading
 to total protonic reversal. That's bad.
 
+Exception: For a clock divider blocking assingment must be used not to case race condition.
```

Comment:
Suggesting:

```
Exception: for a clock divider, blocking assignment must be used to not cause race condition.
```

That said, I'm not agreeing with it yet, and we'll discuss some more internally.

Reply:
Also, I really think this needs different terminology: a race condition causes non-deterministic behaviour, depending on the whims of the simulator's scheduling. Your example has deterministic behaviour (I think), but not the behaviour you want.

Reply:
@sjgitty and @rswarbrick ,
Thank you for your comments.

How about this?

> Exception: for a clock divider, blocking assignment must be used to have the behavior you want.  See #44 for details.

I'm not sure if a link to this pull request is allowed.  But it is too much to include explanation above in the Style Guide.

> That said, I'm not agreeing with it yet, and we'll discuss some more internally.

Thank you.